cypress/e2e/home.cy.ts

```diff
@@ -77,3 +83,42 @@ describe(" Navbar: UI and functionality:", () => {
     cy.get("@themeButtonLight").should("not.exist");
   });
 });
+describe(" Footer: UI and functionality:", () => {
+  it("Should the footer container be visible", () => {
+    cy.get(".footer-container").should("exist").and("be.visible");
+  });
+
+  it("Should the github logo be visible and have a href", () => {
+    cy.get(".github-icon")
+      .should("be.visible")
+      .and("have.attr", "href", githubUrl);
+  });
+
+  it("Should the terms and services and have a href", () => {
```

Comment:
Same here and the ones from below.

Reply:
Fix in commit e46bc72621e2e6b5b3167b81e5de57d91c8b23d3.